src/js/components/useToast.ts

```diff
@@ -27,112 +27,141 @@ import {Toast} from 'bootstrap';
+  return toastObject;
 };
 
-const getToastElement = (template?: string): HTMLElement => {
+const getToastElement = (template?: string): HTMLElement | undefined => {
   const toastContainer = document.querySelector<HTMLElement>(selectorsMap.toast.container);
+  // If the toast container exists on the page, use it
+  // Otherwise, use the fallback container
 
-  // If the toast container exist on the page, use it
-  // Overwise, use the fallback
   if (toastContainer) {
-    const overrideContainer = document.createElement('div');
-    let toastElement;
-
-    if (template) {
-      overrideContainer.innerHTML = template;
-      toastElement = cloneToastTemplate(toastContainer, overrideContainer);
-    } else {
-      toastElement = cloneToastTemplate(toastContainer);
-    }
+    // If override template is set, append to the container
+    // Otherwise, try to clone toast template
 
-    // If the template exist on the page, use it
-    // Overwise, use the fallback
-    if (toastElement) {
-      return toastElement;
-    }
-
-    return useFallbackToastTemplate(toastContainer);
+    return (template === undefined)
+      ? cloneToastTemplate(toastContainer)
+      : appendToastTemplate(toastContainer, template);
   }
 
   return useFallbackToastContainer(template);
 };
 
 // We need to use a template, in order to generate the toast markup on the fly
-const cloneToastTemplate = (containerToAppend: HTMLElement, toastContainer?: HTMLElement): HTMLElement | false => {
-  let toastTemplate;
-
-  if (toastContainer) {
-    toastTemplate = toastContainer.querySelector<HTMLTemplateElement>(selectorsMap.toast.template);
-  } else {
-    toastTemplate = containerToAppend.querySelector<HTMLTemplateElement>(selectorsMap.toast.template);
-  }
-
+const cloneToastTemplate = (toastContainer: HTMLElement, fallback = true): HTMLElement | undefined => {
+  const toastTemplate = toastContainer.querySelector<HTMLTemplateElement>(selectorsMap.toast.template);
   const toastClone = toastTemplate?.content.cloneNode(true) as DocumentFragment;
   const toastElement = toastClone?.querySelector<HTMLElement>(selectorsMap.toast.toast);
   const toastBody = toastElement?.querySelector<HTMLElement>(selectorsMap.toast.body);
 
   if (toastElement && toastBody) {
-    containerToAppend.appendChild(toastElement);
+    toastContainer.appendChild(toastElement);
 
     return toastElement;
   }
 
-  return false;
+  if (fallback) {
+    // In case the template doesn't exist on the page, rely on the fallback
+    const fallbackContainer = getFallbackContainer();
+
+    if (fallbackContainer) {
+      toastContainer.innerHTML = fallbackContainer.innerHTML;
+
+      return cloneToastTemplate(toastContainer, false);
+    }
+  }
+
+  // This happens only if someone removed the markup AND removed the fallback
+  printConsoleError(
+    'Failed to clone toast template.',
+    'Check the toast template in theme markup or JS fallback.',
+  );
+
+  return undefined;
```

Comment:
```suggestion
  return;
```

It will do the same I think so